templates/index.md.tmpl

```diff
@@ -32,6 +32,57 @@ The changelog for this provider can be found here: <https://github.com/grafana/t
 
 {{ .SchemaMarkdown | trimspace }}
 
+### Managing Connections
+
+#### Obtaining Connections access token
+
+Before using the Terraform Provider to manage Grafana Connections resources, such as metrics endpoint scrape jobs, you need to create an access policy token on the Grafana Cloud Portal. This token is used to authenticate the provider to the Grafana Connections API.
+[These docs](https://grafana.com/docs/grafana-cloud/account-management/authentication-and-permissions/access-policies/authorize-services/#create-an-access-policy-for-a-stack) will guide you on how to create
+an access policy. The required permissions, or scopes, are `integration-management:read`, `integration-management:write` and `stacks:read`.
+
+Also, by default the Access Policies UI will not show those scopes, to find name you need to use the `Add Scope` textbox, as shown in the following image:
```

Comment:
"to find name" reads a bit odd to me? (not a native speaker)

Reply:
Updated grammer